assignments/this.js

```diff
@@ -13,14 +13,60 @@
 
 // code example for Window Binding
 
+function giveAge(age){
+    console.log(this);
+    return age;
+}
+giveAge(98);
+
 // Principle 2
 
 // code example for Implicit Binding
 
+    myObj={
+        callOut: 'Over here!',
+        sayName: function(name) {
```

Comment:
This one is a little off. You want to declare `name` as a key on the object and then use it.